server/report/report-service.js

```diff
@@ -102,4 +104,32 @@ export default class {
       path: path.join(options.path, filename)
     });
   }
+  remapSourceMaps(options) {
+    const remapIstanbul = Npm.require('remap-istanbul');
+    const cwd = process.cwd();
+    process.chdir(Conf.COVERAGE_APP_FOLDER);
+
+    // Create output directory if not exists
+    let remapFolder = `${Conf.COVERAGE_EXPORT_FOLDER}-remap`;
```

Comment:
Instead of a separate folder, why not a different file name instead in the regular folder?